ui/nSiteNVar_ui.R

```diff
@@ -106,33 +118,25 @@ nSiteNVar_tab = tabPanel("Multisite", value='multisite_exploration',
                         dygraphOutput("GRAPH_MAIN3a", height='125px'),
                         br()
                     ),
-                    conditionalPanel('input.SOLUTES3.length > 1',
-                    # conditionalPanel('false',
+                    conditionalPanel('input.VARS3.length > 1',
                         div(id='main3b'),
                         dygraphOutput("GRAPH_MAIN3b", height='125px'),
                         br()
                     ),
-                    conditionalPanel('input.SOLUTES3.length > 2',
-                        # fluidRow(
-                        #     column(10, offset=1,
-                                div(id='main3c'),
-                        #     ),
-                        #     column(1,
-                        #         actionButton('EXPAND_MAIN3c', icon('external-link-alt'))
-                        #     )
-                        # ),
+                    conditionalPanel('input.VARS3.length > 2',
```

Comment:
Just another super nitpicky naming question .Are you using dots here because these are directly shiny names and you are saving the snake_case for direct R data programmy things things?

Reply:
Naw, this is javascript! In conditionalPanels, the condition must be specified in native js. And the shiny inputs can be called out from the `input` list via dots.